### PR TITLE
TEAMNADO-4642 - Add create activation key wizard

### PR DESCRIPTION
**Pull request by lukegrn**
# Description

Associated Jira ticket: TEAMNADO-4642

This change adds a new create wizard for activation keys. It is currently locked behind a feature flag and will not be available in prod.

# Checklist:

- [ x] The commit message has the Jira ticket linked
- [ x] PR has a short description

### CODE REVIEW

**Review by leSamo — CHANGES_REQUESTED** [review]
LGTM overall. Matches mockup very closely. I found one bug -- activation key name in first step is marked as a required field but there's nothing preventing me from continuing the wizard when I leave the field empty and then I receive error after clicking `Create` button. Also when I create a key which already exists I get a cryptic `Something went wrong` error notification. It might be better to prevent user from continuing to the next step or at least list that as one of the reasons why it didn't go through in the error notification.

I'd prefer `CreateActivationKeyWizard.js` separated into multiple files as it contains just way to many components and it's hard to read, but maybe you are used to such files in this project idk :smile:

**Comment by lukegrn** [comment]
> LGTM overall. Matches mockup very closely. I found one bug -- activation key name in first step is marked as a required field but there's nothing preventing me from continuing the wizard when I leave the field empty and then I receive error after clicking `Create` button. Also when I create a key which already exists I get a cryptic `Something went wrong` error notification. It might be better to prevent user from continuing to the next step or at least list that as one of the reasons why it didn't go through in the error notification.
> 
> I'd prefer `CreateActivationKeyWizard.js` separated into multiple files as it contains just way to many components and it's hard to read, but maybe you are used to such files in this project idk smile

The lack of validation on the required name field is intended. The story specifies that this is just the happy-path creation flow. This is locked behind a feature flag, and so it will be iterated upon to add error cases, validation, and more error messages.

I also did consider splitting up that file because it is quite large, but decided against it because all of those components function only in the context of the wizard so it doesn't make sense to treat them as standalone components to me. I am happy to make this change though if you think that would be better.

File: src/Components/ActivationKeys/__tests__/ActivationKeys.test.js [hunk 1]
```diff
@@ -65,6 +67,8 @@ jest.mock(
   () => () => <div>Unavailable</div>
 );
 
+useFeatureFlag.mockReturnValue(true);
```
Comment (jmbuscemi) [inline src/Components/ActivationKeys/__tests__/ActivationKeys.test.js:70]: does this set the value to always true even if we turn off the feature flag one day?

Comment (jmbuscemi) [reply src/Components/ActivationKeys/__tests__/ActivationKeys.test.js:70]: ignore me...this is a test file. byeeeee

**Review by leSamo — APPROVED** [review]
Thanks for the clarification! Good to go from my side.

**Review by cowanrc — APPROVED** [review]
LGTM

**Pull request merged**